docs/setting-up.md

```diff
@@ -14,6 +14,7 @@ The instructions in all parts of this document work for Linux, OS X, and Windows
 
 1. Install Source Tree or other similar Git Client, or at least Git.
 1. Install JDK 1.8.
+1. Install Python 2.7.
```

Comment:
Can this be `2.7 or later`?

Reply:
Python 2.7 is the latest of the 2.x version. In fact they explicitly request for a minor version [here](https://cloud.google.com/sdk/docs/). I won't count too much on them supporting Python 3.